src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -13264,6 +13260,92 @@ namespace SurfaceGeometry {
         Surface(SurfNum).Width = WidthEff;
         Surface(SurfNum).Height = HeightEff;
     }
+    
+
+    void CheckForReversedLayers(bool &RevLayerDiffs,    // true when differences are discovered in interzone constructions
```

Comment:
@mjwitte Yeah, had done that early on and ran into some sort of weird problems.  I don't remember what the issue was anymore, but doing this resolved the issue.  My preference would have been a bool function, too.  I'll see what shakes out with the other comments...

Reply:
@mjwitte Now I remember...it was some weird complaint about setting the value of a bool function and then resetting it.  I also found the logic here a bit more complex than a simple set it and return immediately.  So, I did this because of the complexity.  I'd like to leave this as is if you don't mind.  I will fix a couple of the other things though (see other comments).